packages/@react-aria/overlays/docs/PortalProvider.mdx

```diff
@@ -0,0 +1,120 @@
+{/* Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/overlays';
+import {HeaderInfo, PropTable, FunctionAPI, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-aria/overlays/package.json';
+
+---
+category: Utilities
+keywords: [overlays, portals]
+---
+
+# PortalProvider
+
+<PageDescription>{docs.exports.UNSTABLE_PortalProvider.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['UNSTABLE_PortalProvider', 'useUNSTABLE_PortalContext']} />
+
+## Introduction
+
+`UNSTABLE_PortalProvider` is a utility wrapper component that can be used to set where components like
+Modals, Popovers, Toasts, and Tooltips will portal their overlay element to. This is typically used when
+your app is already portalling other elements to a location other than the `document.body` and thus requires
+your React Aria components to send their overlays to the same container.
```

Comment:
We do not address the prefix at all. We should. Maybe:
> The PortalProvider is considered UNSTABLE because there are many places that an application could portal to, and not all of them will work, either with styling, accessibility, or any of a number of reasons. It is best to portal to the root of the entire application, typically the `body` element, outside of any possible overflow or stacking contexts. We envision this typically being used to group all of the portalled elements into a single container at the root of the app or to control the order of children of the `body` element.